test/services-view.spec.ts

```diff
@@ -90,7 +98,68 @@ describe('servicesView', () => {
     expect(jQuery(element).find('.services-item-name').length).toBe(1);
   });
 
+  it("should categorize items correctly when main category has 'tags'", () => {
+    // remove any dynamically added 'Other' categories
+    removeOtherCategories(Constants.SERVICE_CATALOG_CATEGORIES);
```

Comment:
The unit test's before() categorizes using mock items and dynamically adds 'Other' categories where needed.  Need to remove those before categorizing using the items specific to this unit test.